engine/src/main/resources/assets/schemas/schemaCollisionMesh.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Physical configuration of a collision mesh, used for in-game collision physics.",
+  "type": "object",
+  "properties": {
+    "rigidBodies": {
+      "description": "Model representing the collision bounds of an object. Intended to be exported from utility applications",
+      "type": "array",
+      "items": {
+        "properties": {
+          "name": {
+            "type": "string",
+            "description": "Identifying id of this collision mesh.",
+            "pattern": "^\\w+:\\w+$"
+          },
+          "origin": {
+            "description": "The center reference point for the mesh.",
```

Comment:
Can we mention the fact that this is normalized, somewhere? Something like
`The normalized coordinates of the reference point for the mesh.`

Reply:
I have updated the description to reflect this information.